feat: add new category, migrations (#110)
vitejs/awesome-vite: README.md

- [vite-plugin-ruby](https://github.com/ElMassimo/vite_rails/tree/main/package) - Vite configuration for Ruby backends.
- [vite_rails](https://github.com/ElMassimo/vite_rails) - Manage your Rails assets with Vite.

> ## Migration from others
> 
> ### vue-cli
> 
> - [vue-cli-plugin-vue](https://github.com/IndexXuan/vue-cli-plugin-vite) - Use vite today, with vue-cli with minimize codebase modifications.

Reviewer: ```suggestion
## Migrations

### Vue CLI

- [vue-cli-plugin-vue](https://github.com/IndexXuan/vue-cli-plugin-vite) - Use Vite on Vue CLI with minimize codebase modifications.
```
